app/assets/stylesheets/application.scss

```diff
@@ -45,15 +45,11 @@ div.code-block {
   @extend %site-width-container;
   padding-bottom: $gutter * 4;
 
-  > h1 {
+  h1 {
```

Comment:
I believe this direct decedent selector was needed to stop this interfering with another component. Can't remember which one though, possibly the title one.